.github/CODEOWNERS

```diff
@@ -330,6 +330,9 @@ x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json @elastic/kib
 # Security Intelligence And Analytics
 /x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules @elastic/security-intelligence-analytics
 
+# Security Asset Management
+/x-pack/plugins/osquery @elastic/security-asset-management
```

Comment:
Hmm, why isn't @elastic/security-asset-management being requested for review...

Reply:
@spalger because those changes are not merged yet I believe

Reply:
Hmm, perhaps it does rely on the base branch, I'm also unable to select that group for review manually

Reply:
I guess what I mean to imply is that I'm not reviewing this entire PR with this change and someone else from that group should probably be requested.

Reply:
you're right, seems the team needs to be assigned to the kibana project, do you know maybe who should I speak to have it done?

Reply:
Thank you @spalger for the help! :)